File: 30-days-of-code/Day06/Review.java
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   Review.java                                        :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/02/25 21:17:31 by alaamimi          #+#    #+#             */
/*   Updated: 2021/02/25 21:17:44 by alaamimi         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

import java.util.Scanner;

public class Solution
{

	public static void main(String[] args)
	{
		Scanner scanInput = new Scanner(System.in);
		int n = scanInput.nextInt();
		scanInput.nextLine();
		int index;

		index = 0;
		while (index < n)
		{
			String str = scanInput.nextLine();
			char[] charArray = str.toCharArray();
			index++;
			int i;

			i = 0;
			while(i < charArray.length)
			{
				if (i % 2 == 0)
				{
					System.out.print(charArray[i]);
				}
				i++;
			}
			System.out.print(" ");
			int j;

			j = 0;
			while (j < charArray.length)
			{
				if (j % 2 == 1)
				{
					System.out.print(charArray[j]);
				}
				j++;
			}
			 System.out.println();
		}
		scanInput.close();
	}
}

File: 30-days-of-code/Day06/Review.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    Review.py                                          :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/03/06 17:32:56 by alaamimi          #+#    #+#              #
#    Updated: 2021/03/06 18:25:27 by alaamimi         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

#!/bin/bash 

T = int(input())

for i in range(T):
    string = str(input())
    even = ""
    odd = ""

    for j in range(len(string)):
        if j % 2 == 0:
            even += string[j]
            else:
                odd += string[j]
    print('{} {}'.fotmat(even, odd))


